src/pages/[visType]/index.tsx

```diff
@@ -134,4 +135,17 @@ PropagatedPageList.getLayout = function getLayout(page: ReactElement) {
   return <DashboardLayout>{page}</DashboardLayout>;
 };
 
+export const getStaticPaths: GetStaticPaths = async () => {
+  return {
+    paths: ["/dashboard", "/plot", "/analytics", "/model"],
```

Comment:
@saifulkhan are these all vis types we need? I complied the list by looking at the side menu.

If the number of vis types is static, we are good to go.

If the number of vis types is dynamic but changes rarely, we can make an API request in this function and thus still hardcode the list on `yarn build`.

If the number of vis types is dynamic and changes frequently, we can change the route to something like `list-pages?visType=dashboard` and thus have to build only one server-side route.

Reply:
Looks good. Thanks.